composer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name" : "zohocrm/php-sdk",
 	"description" : "Zoho CRM API SDK for PHP",
-	"type" : "SDK",
```

Comment:
Composer 1.9.1 complained that `type` must be lower-case.

```
  [Composer\Json\JsonValidationException]
  "./composer.json" does not match the expected JSON schema:
   - type : Does not match the regex pattern ^[a-z0-9-]+$
```

Composer seems to recommend leaving this unset or using `library`.
https://getcomposer.org/doc/04-schema.md#type